src/components/ModalPopup/ModalChatPayment.tsx

```diff
@@ -65,6 +71,10 @@ const ModalChatSettlement = ({
       method: "post",
       data: { roomId: roomInfo._id },
       onSuccess: () => {
+        //#region event2023Fall
+        event2023FallQuestCompletePS();
+        event2023FallQuestCompleteP();
+        //#endregion
```

Comment:
아까 퀘스트 아이디를 훅이 반환하는 함수의 파라미터로 옮기면 이거 더 간단해집니다.

```suggestion
        //#region event2023Fall
        event2023FallQuestComplete("payingAndSending");
        event2023FallQuestComplete("paying");
        //#endregion
```

Reply:
오 훨씬 나은 것 같아요